plugins/webp-uploads/helper.php

```diff
@@ -88,11 +88,19 @@ function webp_uploads_generate_additional_image_source( $attachment_id, $image_s
 	 *
 	 * @since 1.1.0
 	 *
-	 * @param array|null|WP_Error $image         Image data {'path'=>string, 'file'=>string, 'width'=>int, 'height'=>int, 'mime-type'=>string} or null or WP_Error.
```

Comment:
Note that `width`, `height`, and `mime-type` are actually _not_ used here. The only keys used are `file` and either `path` or `filesize`.